ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py

```diff
@@ -8,3 +10,20 @@ def ifft(a, n=None, axis=-1, norm=None):
     if norm is None:
         norm = "backward"
     return ivy.ifft(a, axis, norm=norm, n=n)
+
+
+
+def fft2_layer(
+        x: np.ndarray,
+        s: Optional[Tuple[int, int]] = None,
+        axes: Tuple[int, int] = (-2, -1),
+        norm: Optional[str] = None) -> np.ndarray:
+    if norm is None:
+        norm = "backward"
+    return np.fft.fft2(x, s=s, axes=axes, norm=norm)
+
+@to_ivy_arrays_and_back
+def fft2(a, s=None, axes=(-2, -1), norm=None):
+    a = ivy.array(fft2_layer(a, s=s, axes=axes, norm=norm), dtype=ivy.complex128)
```

Comment:
you may either complete this function as a composition of other functions, or in other cases, propose the `fft2` function to be added to the ivy functional API if you see fit, more information here in the section! 
https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis